addon/components/yeti-table/component.js

```diff
@@ -273,15 +293,20 @@ class YetiTable extends DidChangeAttrsComponent {
     }
   }
 
-  @computed('loadData', 'sortedData.[]', 'resolvedData.[]')
+  @computed('loadData', 'onFilteredRowsChanged', 'resolvedData.[]', 'sortedData.[]')
   get normalizedRows() {
+    let rows;
     if (!this.get('loadData')) {
       // sync scenario using @data
-      return this.get('sortedData');
+      rows = this.get('sortedData');
     } else {
       // async scenario. @loadData is present.
-      return this.get('resolvedData');
+      rows = this.get('resolvedData');
     }
+    if (this.onFilteredRowsChanged) {
+      this.onFilteredRowsChanged(rows);
```

Comment:
It is considered an anti-pattern to have side-effects on computed properties.
Computed properties (js getters) as supposed to be pure functions.

Instead, we should use the `didChangeAttrsConfig` hook to listen for changes on `normalizedRows`.
You can configure it to run on `normalizedRows` changes with:
```
this.didChangeAttrsConfig = {
  attrs: ['filter', 'filterUsing', 'pageSize', 'pageNumber', 'normalizedRows']
};
```

This is on the `init()` hook.

Reply:
didChangeAttrs only observes parameters that were passed into the component and I don't think normalizedRows would work because it is not passed in. 

However, we could observe the filteredRows using filter and filterUsing so we could intercept it here.